website/docs/getting-started/setup/installation-guides/kubernetes/README.mdx

```diff
@@ -11,72 +11,99 @@ This page provides steps to install Appsmith on a Kubernetes cluster using the [
+2. Installed and configured `kubectl` to interact with your Kubernetes cluster. Follow the below guides available on the official Kubernetes documentation for instructions on how to install `kubectl` on your specific operating system: 
+    * [MacOS using Homebrew](https://kubernetes.io/docs/tasks/tools/install-kubectl-macos/#install-with-homebrew-on-macos)
+    * [Linux using native package management](https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/#install-using-native-package-management)
+    * [Windows using Chocolatey, Scoop, or winget](https://kubernetes.io/docs/tasks/tools/install-kubectl-windows/#install-nonstandard-package-tools).
+3. A running Kubernetes cluster. If not present, follow these steps to set up the cluster on AWS EKS:
+
+  :::info
+   You are free to choose your preferred platform for hosting the Kubernetes cluster. Make sure you have configured Kubeconfig and created a default storage class before proceeding with the Appsmith installation. If you face any issues, reach out to the support team at [support@appsmith.com](mailto:support@appsmith.com).
+  :::
+
+    a. Verify if you have access to AWS CLI with:
+      
+    ```bash
+      aws sts get-caller-identity 
+    ```
+    
+    The above command provides information related to your account and ARN, indicating that you can connect and access your Amazon account using a terminal.
 
-</TabItem>
-<TabItem label="Google Cloud Kubernetes" value="GoogleCloudKubernetes">
+    b. Create KubeConfig with: 
 
-  a. [Configure cluster access for kubectl](https://cloud.google.com/kubernetes-engine/docs/how-to/cluster-access-for-kubectl) on Google Cloud Kubernetes.
+    ```bash
+    aws eks update-kubeconfig --region ap-south-1 --name CLUSTER_NAME  --profile PROFILE_NAME
+    ```
+    In the above command, add the profile name that has access to the EKS cluster to the `--profile` parameter.
 
-  b. Ensure that you have a default storage class running. If not, [set up a default storage class on Google Kubernetes Engine](https://cloud.google.com/anthos/clusters/docs/on-prem/1.3/how-to/default-storage-class)[GKE].
+    c. Test your Kubernetes configuration with:
 
-</TabItem>
-<TabItem label="Microk8s" value="Microk8s">
+    ```bash
+    kubectl cluster-info
+    ```
+    The above command provides a summary of the current cluster configuration, including the Kubernetes master and other cluster information. 
 
-  a. Set up Kubernetes by [working with kubectl](https://microk8s.io/docs/working-with-kubectl).
+    d. A default storage class running. If not created yet, create the storage class with:
 
-  b. Ensure that you have a default storage class running. If not, [enable storage class](https://microk8s.io/docs/command-reference#heading--microk8s-enable).
-    
-</TabItem>
-</Tabs>
+    ```bash
```

Comment:
if we are providing them commands, we should always specify where they have run these.

Reply:
Just looked at it again, all the above steps are commands only, and they don't need to do it in a specific folder. Just on the terminal. I think it's intuitive that the users will open a terminal and we don't have to specify that. @pratapaprasanna - Please confirm.